conf/ocsci/rosa.yaml

```diff
@@ -0,0 +1,35 @@
+---
+ENV_DATA:
+  platform: 'rosa'
+  deployment_type: 'managed'
+  addon_name: 'ocs-converged'
+  configs: 'aws,stage'
+  worker_instance_type: 'm5.2xlarge'
+  worker_replicas: 3
+  master_replicas: 3
+  # size of storage cluster, default is 1 TiB
+  size: '1'
+  #skip_ocp_deployment: true
+  # notification_email_0: MAIL USED FOR SENDGRID
+
+# AUTH:
+#  openshiftdedicated:
+#    token: OCM TOKEN KEY PLACEHOLDER
+
+DEPLOYMENT:
+  ocs_csv_channel: "latest-stable-4.8"
```

Comment:
shouldn't we put the version in a separate yaml?
The same convension as in ocs_ci/framework/conf/ocs_version/ocs-4.x.yaml